### PR TITLE
fix for issue #5810 and update java mission

### PR DESCRIPTION
**Pull request by iwonahahn**
@thecodester and @MichaelCzcz : Please wait with appoval and merging, dev colleagues are still testing the changes. Will comment here once we can merge. Thanks a lot.

### CODE REVIEW

File: tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ cd ~/projects
 3. Now that you are in the correct folder, run the following command:
 ```Shell/Bash
 mvn -B archetype:generate -DarchetypeArtifactId=cds-services-archetype -DarchetypeGroupId=com.sap.cds \
--DarchetypeVersion=1.3.0 -DcdsVersion=3.31.2 \
+-DarchetypeVersion=Release \
```
Comment (beckermarc) [inline tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md:40]: please change `Release` to `RELEASE`

File: tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ You'll now define the services, that should expose the entities you've defined i
     // Define Orders Service
     service OrdersService {
         entity Orders as projection on db.Orders;
-        // OrderItems are auto exposed
+        // entity OrderItems as projection on db.OrderItems;
```
Comment (beckermarc) [inline tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md:174]: Please remove the comment indicator `//`

File: tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md (hunk 1 above)
Comment (beckermarc) [reply tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md:40]: The same change also needs to be applied to this file: https://github.com/SAPDocuments/Tutorials/blob/master/tutorials/cp-cap-java-app-studio/cp-cap-java-app-studio.md

File: tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md [hunk 3]
```diff
@@ -262,7 +262,7 @@ npm install --save-dev sqlite3
 
 3. To initialize the bookstore database with the defined domain model and sample data, run:
 ```Shell/Bash
-npm run deploy
+cds deploy --to sqlite
```
Comment (beckermarc) [inline tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md:265]: This file contains the same string once again, in step 7, bullet point 3. Please also replace it there.
In addition these two files also need to be adapted with that change:
- https://github.com/SAPDocuments/Tutorials/blob/master/tutorials/cp-cap-java-custom-logic/cp-cap-java-custom-logic.md (once)
- https://github.com/SAPDocuments/Tutorials/blob/master/tutorials/cp-cap-java-reusable-service/cp-cap-java-reusable-service.md (twice)
- https://github.com/SAPDocuments/Tutorials/blob/master/tutorials/cp-cap-java-reusable-service/run-deploy.png (screenshot)

File: tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md (hunk 1 above)
Comment (iwonahahn) [reply tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md:40]: will do the changes in a new pr

**Pull request merged**